debian/libwobbly-glib0.install

```diff
@@ -0,0 +1 @@
+usr/lib/*/libwindowfx_wobbly_glib.so*
```

Comment:
Usually the `usr/lib/*/*.so` file goes in the -dev package, while the versioned `.so.*` go in the non-dev one.

Reply:
Useful to know